### PR TITLE
Misc improvements to code review tool

### PR DESCRIPTION
**Pull request by marco-c**
Each commit is an isolated change.

### CODE REVIEW

File: scripts/code_review_tool_evaluator.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def get_file(commit_hash, path):
             (
                 "RAG",
                 code_review.CodeReviewTool(
-                    llm=llm,
+                    comment_gen_llms=[llm],
```
Comment (suhaibmujahid) [inline scripts/code_review_tool_evaluator.py:75]: Why do we pass the llm as a list here?

File: bugbug/tools/code_review.py [hunk 2]
```diff
@@ -1074,11 +1082,17 @@ def run(self, patch: Patch) -> list[InlineComment] | None:
...
+        if self.verbose:
+            print(f"\u001b[33;1m\033[1;3m{output_summarization}\u001b[0m")
+
         if self.function_search is not None:
             line_code_list = self.further_context_chain.run(
                 patch=formatted_patch, summarization=output_summarization
             ).split("\n")
 
+            if self.verbose:
+                print(f"\u001b[33;1m\033[1;3m{line_code_list}\u001b[0m")
```
Comment (suhaibmujahid) [inline bugbug/tools/code_review.py:1094]: We may rap the print formatting in a function to make it more readable.

**Review by suhaibmujahid — APPROVED** [review]
LGTM!

File: scripts/code_review_tool_evaluator.py (hunk 1 above)
Comment (marco-c) [reply scripts/code_review_tool_evaluator.py:75]: The idea would be to support an ensemble of LLMs (either the same model with different temperature and top_p, or different models) to generate diverse comments, and then deduplicate the comments.

Comment (suhaibmujahid) [reply scripts/code_review_tool_evaluator.py:75]: We may have such implication details in a class that uses the LLM interface. So this class doesn't need to know how the comments were generated.

File: bugbug/tools/code_review.py (hunk 2 above)
Comment (marco-c) [reply bugbug/tools/code_review.py]: Done!

File: scripts/code_review_tool_evaluator.py (hunk 1 above)
Comment (marco-c) [reply scripts/code_review_tool_evaluator.py:75]: Good idea, filed #4539 as a follow-up to do this.

**Pull request merged**